src/Content/Widget/VCard.php

```diff
@@ -119,6 +124,10 @@ public static function getHTML(array $contact): string
 			'$unfollow_link'    => $unfollow_link,
 			'$wallmessage'      => DI::l10n()->t('Message'),
 			'$wallmessage_link' => $wallmessage_link,
+			'$mentioning'       => DI::l10n()->t('Mention'),
+			'$post2group'       => DI::l10n()->t('Post to group'),
```

Comment:
Please rename closer to the target label.
```suggestion
			'$mention'          => DI::l10n()->t('Mention'),
			'$posttogroup'      => DI::l10n()->t('Post to group'),
```

Reply:
changed it.